README.rst

```diff
@@ -18,7 +18,143 @@ SpringField is ideal for:
 - Parsing CSV data structures from ``csv.DictReader``
 - Turning anything Python can parse into a ``dict`` or ``list`` into a structured object
 
+There is also a helper library for using SpringField with Mongo: `springfield-mongo <https://github.com/six8/springfield-mongo>`_
+
+
+Quickstart
+----------
+
+To define an ``Entity``, subclass ``springfield.Entity``. Define your attributes by specifying
+``fields``. This library provides the follow self-describing fields to start with:
+
+- ``IntField``
+- ``FloatField``
+- ``BooleanField``
+- ``StringField``
+- ``BytesField``
+- ``DateTimeField``
+- ``EmailField``
+- ``UrlField``
+- ``EntityField``
+- ``CollectionField``
```

Comment:
How can I do that? I mean what is the path/href I should use? I could link to src (`src/springfield/entity.py`), to the RST (`docs/entity.rst`) or to something else (maybe RTD html link?) but I'm not sure what it should be. Ideally I'd change this to link directly to the anchor of the definitions, but I just don't know how.